examples/0_Simple/vectorAdd_lib/host.gpr

```diff
@@ -0,0 +1,41 @@
+------------------------------------------------------------------------------
+--                       Copyright (C) 2017, AdaCore                        --
+-- This is free software;  you can redistribute it  and/or modify it  under --
+-- terms of the  GNU General Public License as published  by the Free Soft- --
+-- ware  Foundation;  either version 3,  or (at your option) any later ver- --
+-- sion.  This software is distributed in the hope  that it will be useful, --
+-- but WITHOUT ANY WARRANTY;  without even the implied warranty of MERCHAN- --
+-- TABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public --
+-- License for  more details.  You should have  received  a copy of the GNU --
+-- General  Public  License  distributed  with  this  software;   see  file --
+-- COPYING3.  If not, go to http://www.gnu.org/licenses for a complete copy --
+-- of the license.                                                          --
+------------------------------------------------------------------------------
+
+with "../../../api/cuda_host.gpr";
+
+project Host is
+
+   for Exec_Dir use ".";
+   for Object_Dir use "obj/host";
+   for Source_Dirs use ("src/common", "src/host");
+   for Main use ("main.adb");
+
+   for Create_Missing_Dirs use "True";
+
+   package Compiler is
+      for Switches ("ada") use ("-gnatX", "-g", "-gnatd_c", "-gnatDGL");
```

Comment:
Argh, this is a leftover from my examples and isn't actually needed (`-gnatDGL` makes GNAT emit its expansion code, which is useful for debugging GNAT but not much else).
```suggestion
      for Switches ("ada") use ("-gnatX", "-g", "-gnatd_c");
```